.NET/Microsoft.Recognizers.Text.Number/Japanese/Parsers/JapaneseNumberParser.cs

```diff
@@ -0,0 +1,495 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Recognizers.Text.Number.Japanese
+{
+    public class JapaneseNumberParser : BaseNumberParser
+    {
+        protected new readonly JapaneseNumberParserConfiguration Config;
+
+        public JapaneseNumberParser(JapaneseNumberParserConfiguration config) : base(config)
+        {
+            this.Config = config;
+        }
+
+        public override ParseResult Parse(ExtractResult extResultJpn)
```

Comment:
Is all the logic in this method new? Or is it mostly the same as Chinese?
Can't more code be reused?